CHANGELOG.md

```diff
@@ -172,6 +172,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 Initial release.
 
 [unreleased]: https://github.com/contentful/contentful-management.java/compare/cma-sdk-3.3.1...HEAD
```

Comment:
Here we also need to update the `3.3.1` to `3.3.2`.

Reply:
Good catch